AzureTable/Microsoft.DataTransfer.AzureTable/RemoteLogging/RemoteLoggingClientProvider.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Concurrent;
+using Microsoft.Azure.CosmosDB.Table;
+using Microsoft.Azure.Storage;
+
+namespace Microsoft.DataTransfer.AzureTable.RemoteLogging
+{
+    /// <summary>
+    /// Provides a client for remote logging in the application.
+    /// </summary>
+    public class RemoteLoggingClientProvider
+    {
+        private static readonly ConcurrentDictionary<string, RemoteLogging> clientProvider =
+            new ConcurrentDictionary<string, RemoteLogging>();
+
+        /// <summary>
+        /// If a remote logger exists, provide it to the caller
+        /// </summary>
+        /// <param name="key"></param>
```

Comment:
Please make sure xmldoc is provided throughout the whole PR. There are multiple places where params and returns are missing.

Reply:
noted. updated xmldocs in all the new code added in this pr.